apps/network-alerts/test/unit/attention/attention_test.js

```diff
@@ -81,103 +79,125 @@ suite('Network Alerts - Attention Screen', function() {
     navigator.mozApps = realMozApps;
   });
 
-  test('form is properly displayed', function() {
-    assert.equal(
-      document.querySelector('h1').getAttribute('data-l10n-id'),
-      title,
-      'The title is properly displayed'
-    );
-    assert.equal(
-      document.querySelector('p').textContent, body,
-      'The body is properly displayed'
-    );
-  });
+  suite('display from received message', function() {
```

Comment:
I moved a lot of tests inside this suite, to make things cleaner for the case where we display from the notification